src/Subscriptions/Authorizer.php

```diff
@@ -72,4 +75,23 @@ public function authorize(Request $request): bool
             return false;
         }
     }
+
+    /**
+     * Removes the prefix "presence-" from the channel name.
+     *
+     * When connecting to a presence channel named "private-lighthouse-subscription-1234"
+     * Laravel Echo prefixes the channel with "presence-", but we store the channel
+     * without the "presence-" prefix, which is what we have to remove here.
+     */
+    private function sanitizeChannelName(?string $channelName): ?string
```

Comment:
```suggestion
    private function sanitizeChannelName(string $channelName): string
```

Let's handle `null` as early as possible.